testData/generate/Getter1.txt

```diff
@@ -1,7 +1,7 @@
 class Getter1 {
-  var foo(getFoo, null):Int;
+  @:isVar public var foo(get, null):Int;
```

Comment:
So, I guess I'm asking, which one is it going to be?  If we're fixing up the access declaration, -- and correct me if they are called something else -- then we should be /removing/ the getters and setters that you are now declaring private.